src/servers/ZoneServer2016/entities/character.ts

```diff
@@ -1038,6 +1038,7 @@ export class Character2016 extends BaseFullCharacter {
   async damage(server: ZoneServer2016, damageInfo: DamageInfo) {
     if (
       server.isPvE &&
+      this.isRespawning &&
```

Comment:
bad logic it's why the test fail, try running "npm t" to see the issue it create

Reply:
Ahh okay 👍